client/components/RelatedItems/RelatedItems.jsx

```diff
@@ -39,10 +39,6 @@ class RelatedItems extends React.Component {
   getRelatedItemsIds (id) {
     axios.get('/get', {params: {endpoint: `products/${id}/related`}})
       .then((response) => {
-        var storage = JSON.parse(localStorage.getItem('allproducts')) || [];
-        // if there's already data in storage
-          // retrieve it
-        // otherwise
         var promises = [];
         for (var i = 0; i < response.data.length; i++) {
           promises.push(this.getAllProductInfo(response.data[i]));
```

Comment:
I like how you used Promise.all  to resolve the request. I could learn from that instead of nesting all my requests in then statements